pkg/model/config/config.go

```diff
@@ -147,34 +147,53 @@ func (c Config) resourceAPIVersionCompatible(verType, version string) bool {
 	return true
 }
 
-// GVK contains information about scaffolded resources
-type GVK struct {
+// Data contains information about scaffolded resources
+type Data struct {
 	Group   string `json:"group,omitempty"`
 	Version string `json:"version,omitempty"`
 	Kind    string `json:"kind,omitempty"`
 
-	// CRDVersion holds the CustomResourceDefinition API version used for the GVK.
+	// CRDVersion holds the CustomResourceDefinition API version used for the Data.
 	CRDVersion string `json:"crdVersion,omitempty"`
-	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the GVK.
+	// WebhookVersion holds the {Validating,Mutating}WebhookConfiguration API version used for the Data.
 	WebhookVersion string `json:"webhookVersion,omitempty"`
+
+	// Webhooks holds the Webhooks data
+	Webhooks Webhooks `json:"webhooks,omitempty"`
+}
+
+// Webhooks contains information about scaffolded webhooks
+type Webhooks struct {
+	Defaulting bool `json:"defaulting,omitempty"`
+	Validation bool `json:"validation,omitempty"`
+	Conversion bool `json:"conversion,omitempty"`
 }
 
-// isEqualTo compares it with another resource
-func (r GVK) isEqualTo(other GVK) bool {
+// isGVKEqualTo compares it with another resource
+func (r *Data) isGVKEqualTo(other Data) bool {
```

Comment:
The name should be updated to reflect the type rename from `GVK` to `ResourceData`. Also, this method does not modify `r` and thus a non-pointer receiver should be preferred.
```suggestion
// isSameResource compares it with another resource
func (r ResourceData) isSameResource(other ResourceData) bool {
```

Reply:
isGVKEqualTo -> only checks if is the same GKV because of this we cannot change this one. we are no comparing if is the same resource data.

Reply:
Okey I see your point, but please remove the `*` as it doesn't modify the values so no need for a pointer receiver.